src/main/java/seedu/address/logic/parser/GradeTrackerParser.java

```diff
@@ -58,8 +58,8 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddGradeCommand.COMMAND_WORD:
             return new AddGradeParser().parse(arguments);
 
-        case ClearCommand.COMMAND_WORD:
-            return new ClearCommand();
+        case ClearModuleCommand.COMMAND_WORD:
+            return new ClearModuleCommand();
```

Comment:
Actually I don't think this needs to be under grade tracker? Since it would already be caught under module list parser, but there isn't any issue with leaving it here either.

Reply:
Yes I was also thinking the same, I'll remove it then.